src/RolloutScorer/RolloutScorer.Tests/RolloutScorerTests.cs

```diff
@@ -56,8 +58,8 @@ public async Task RedirectApiTest(string requestPath, string responseUri, string
             HttpResponseMessage sameHostRedirectResponse = new HttpResponseMessage();
             sameHostRedirectResponse.Headers.Location = new Uri(responseUri);
 
-            HttpRequestException exception = (await (((Func<Task>)(                async () => await _rolloutScorer.GetAzdoApiResponseAsync(
-                    Utilities.HandleApiRedirect(sameHostRedirectResponse, sameHostRedirectUri))))).Should().ThrowAsync<HttpRequestException>()).Which;
+            HttpRequestException exception = (await ((Func<Task>)(                async () => await _rolloutScorer.GetAzdoApiResponseAsync(
```

Comment:
If you're going to touch this horribly formatted line, might want to clean it up a bit.

This was done by the auto-nunit-ifier.

The second line could also be combined with the first.  Also I don't like asserting about messages. :-)